extensions/amp-analytics/0.1/test/test-analytics-root.js

```diff
@@ -383,43 +384,55 @@ describes.realWin('AmpdocAnalyticsRoot', {amp: 1}, (env) => {
         child.id = 'myId';
         child.classList.add('myClass');
         expect(
-          await root.getAmpElements(body, ['.myClass', '#myId'], null)
+          await root.getElements(body, ['.myClass', '#myId'], null)
         ).to.deep.equal([child]);
       });
 
       it('should ignore special selectors', async () => {
         child.classList.add('myClass');
         expectAsyncConsoleError(/Element ":host" not found/, 1);
         await expect(
-          root.getAmpElements(body, [':host'], null)
+          root.getElements(body, [':host'], null)
         ).to.be.rejectedWith(/Element ":host" not found​​​/);
       });
 
       it('should handle missing selector for AMP search', async () => {
         expectAsyncConsoleError(/Element "#unknown" not found/, 1);
         await expect(
-          root.getAmpElements(body, ['#unknown'], null)
+          root.getElements(body, ['#unknown'], null)
         ).to.be.rejectedWith(/Element "#unknown" not found​​​/);
       });
 
       it('should handle invalid selector', async () => {
         expectAsyncConsoleError(/Invalid query selector 12345/, 1);
-        await expect(
-          root.getAmpElements(body, [12345], null)
-        ).to.be.rejectedWith(/Invalid query selector 12345​​​/);
+        await expect(root.getElements(body, [12345], null)).to.be.rejectedWith(
+          /Invalid query selector 12345​​​/
+        );
       });
 
-      it('should fail if the found element is not AMP for AMP search', async () => {
-        expectAsyncConsoleError(/required to be an AMP element/, 1);
+      it('should find both AMP and non AMP elements', async () => {
         child.classList.remove('i-amphtml-element');
-        await expect(
-          root.getAmpElements(body, ['#child'], null)
-        ).to.be.rejectedWith(/required to be an AMP element/);
+        child.classList.add('myClass');
+        child2.classList.add('myClass');
+        expect(await root.getElements(body, ['.myClass'], null)).to.deep.equal([
+          child,
+          child2,
+        ]);
+      });
+
+      it('should find non AMP element', async () => {
```

Comment:
I think the previous test covers the non AMP element, can we remove this one?

Reply:
The difference with the previous test is that it's using the new array selector. Will make it more clear.